File: examples/sharing/accept-invitation.ts
import type { JsBaoClient } from "js-bao-wss-client";

// Redeem an invite token (resolves all linked deferred grants to the caller).
export async function acceptInvitation(client: JsBaoClient, inviteToken: string) {
  // #region example
  const result = await client.invitations.accept(inviteToken);
  // #endregion example
  return result;
}

File: examples/sharing/accept-invitation.swift
import JsBaoClient

// Redeem an invite token (resolves all linked deferred grants to the caller).
func acceptInvitation(client: JsBaoClient, inviteToken: String) async throws {
  // #region example
  let result = try await client.invitations.accept(inviteToken: inviteToken)
  // #endregion example
  _ = result
}
